logplex.gemspec

```diff
@@ -6,11 +6,11 @@ require 'logplex/version'
 Gem::Specification.new do |gem|
   gem.name          = "logplex"
   gem.version       = Logplex::VERSION
-  gem.authors       = ["Harold Giménez"]
+  gem.authors       = ["Harold Giménez", "Heroku"]
   gem.email         = ["harold.gimenez@gmail.com"]
```

Comment:
should we update this?

Reply:
Maybe a team mailing list or something?

Reply:
Yes, the real question is to what!

Reply:
Given this is a public gem, internal mailing lists are probably not the right call. I thought about `support@heroku.com`.